--- a/honeypot.install	2026-09-26 23:49:15
+++ b/honeypot.install	2026-09-26 23:49:15
@@ -118,9 +118,11 @@
  * Creates an 'id' column for the {honeypot_user} table to act as a primary key.
  */
 function honeypot_update_8102() {
-  $schema = honeypot_schema();
-  $spec = $schema['honeypot_user']['fields']['id'];
-  \Drupal::database()->schema()->addField('honeypot_user', 'id', $spec, ['primary key' => ['id']]);
+  if (!\Drupal::database()->schema()->fieldExists('honeypot_user', 'id')) {
+    $schema = honeypot_schema();
+    $spec = $schema['honeypot_user']['fields']['id'];
+    \Drupal::database()->schema()->addField('honeypot_user', 'id', $spec, ['primary key' => ['id']]);
+  }
 }
 
 /**
